faviconupdate.js

```diff
@@ -1,165 +1,230 @@
ekcRPCmMkrHa+ncRsPqPbS+4fWeiRQICuk0+vrMpGP/KnIdY1lfixX8OwJYJJzy6a8GQoZ0IKq+A5mwryOR/bHPcyhNRH8MT8Q9fG4+LT+OAZ4CoM6NYR1mHG3bO2gfLoh4d164f8MAb5QeIgDmVgTlCvcJ/b2D9sm4OOj9CDq2H4yqe2987Vpq30N/LwelzykUeOv2ZAHBUDoe6U+2hMrreA3Bp+5IBHwqKq9VEe1dHwUgFeYPD4bpDKXHd9NyYHm+TACMVAOuIsed0A2EVkuPeRGyJpbjQzonu0B2I9oJ+RaCQf4qHttjP8bIZ5T+YB4X0wPzvYO2x6oYP4vjVwt2GEHGdTltc77NN0EVOUa2iqCIHMgdHpCaOnk1KM2MePeWiPNkAUI9SuqwuR8AqshfEJjRETVWw1thoOxkjnGcz6V9j3NuBTk+oABI2xHQkwJroU4Z3q0x2s8ymbF5uOYsfMd9A5UFDzNjEVkiHgMVgtpt7OMe8l+0eOL6+Ev7egIzYkLAUATIunJfkgAYI+gXbvT1UsQcD2sMr9Dn1sIApng30Yhh3CG9wrG3iOT1ctQ9O617JZ3nC6mc0ZvMWUl1toXkO5Xk3eJwG51IJ9KJdPyl42m7YvWqqQ74vMyl7byK529s1nlL1HlHfz5Gnnec6N7+pYfNK5gHvOdpSpXi7RnU6Pd+d5Cjr7n36/d5njkBDhNb6vJeemNTW59CHLnXq+XmCPcTSmwfPZnkDhbifdFs5GBMkRU8fhnsVv7ESBqE6bmvG1DZOADdCPP9eQd5wRf06JGePQplLjs/Fw3jGuL3qOX180/04okUl+BDxJpE1k1lVeVlGcJemEWDtR0IP5kmKyoq7+m4wS7hO5zIaJBMuBQO3qCigd6kGnkt0pXRYmFVxRfSIofxaOVYQGP6uhOjKraUO9TUWFvuxbhijukCV1ChrS4AqRkGMFo5yB9uhlbytxxSISVTINeFHsBL2FPBgNxWtcudMy7G/M+rdpdDtVtDYNDSu/oa0h67kXz3+y1KKlGT3/n533vIXAiGhQBYyhGIhJIAEVEUxU8OTpHlSEeOHhhXEilUu1MUPTwDFjYkhYIqE/T7+HSjLHsDCpTXuliRkUPtnlBMxj8N2APrq/QUKrE+eY8c7gMQBsYJ11cvb/erKqMOqv5X+/BeCo97/A1JEvBhx7YBmA5k7p4b9CsgGBhy8Nq8ZvuYd9EAF9j+na0hDJYswPcVWZ14NNpt91rdJjB2tqPH3+PmJkf1x2qyEd5v0e0h9ww2ZSJ+jcnvINDAt45LWkjmLSuazPNmDoMvv94f+/iPolsIahX+bXh4Hijew/2QhLzRkPnz13s8n6Dz2R/DJyaLGWy2FKg6qMUuHBZZzfO8dgAYwp2xkOM4SM/QwJdxHiRl53UkC1aDPueJMLTBwnJjLQKHY6GSZFx0/jg7eCTF9fLXdVfvrXdVMBzrtJkd9St3ycpAd3myR+HE/I0nR1+rFM9jGZ6zko61EE5kVoKSnRVYQWUYdDTDQeEq9n94mD0wGAcuV2mHenIpdjW6cjSE4J/76tgfDc9TtnD+rNwsW2N0v2CaMtw8Pbsx1PTWoUb28sJsBdrJjfYgGTetMNgSmTqX9IliQTEpNeAZSQOjZCKmGg5QE5EJTZrcDhqQ78SjU+jkfSokq+fn2YWY52d4LiSjKAisyBlYyMi1QHONO6ZJ5nMymgM3xCn7ZGfmaPnV1dWC0WAENS4QrQDYLdlxtUVqSuoYe44ZjBYbOMwWaG9uhKa66nstJuOjmnwX1e0onwgsCxNSLI/5TKWP8fY0cPAc5JD6W5OAi4bgVyyC+3paCE1wkqZRgAVJP1QJQu0tEHJ1AqcGtWvJIKgoe2t0ZcaM/gknJG1Zv77HZQzaSYQPUgCrqqL/iRnBooq8m0gLGBNvQY0WcQm8f1ywR0PG2LeLSkvUU85fVL9l/cZPwzK32WKpYQ2pH6FMkpZTZDYYDHsrq5qo/D4gmFBHDfxqycJht08qNOdsO+TtQz/tXqskIrA6MFoQQKfHB59vKovdBichKLgIgzWJBhFy83NB1YAQZa0+Wr3I36gc1tU2Ish79/MmMwyddDrUVNcPSHmkUAj79yB8dxA3OsQd4AWekUJJ2Yj5RqIDI3AnOrbXLHng61+ed924fgFgPf+fn/M/vfCvySjE++S4MhQKwfad22D6ybqvG+oXM6fPAol1ExrfLadUNe+MpPcrnvzxmriZn3oMzSZ+JVgre128P+g3KAbWiGC3tU01jCRLHNtHQKlT9iosMMqWsjLFJBoUSZGDBkb0HWpuDMZRhNPxnywbH+ZFGUhSG1whzc6Esh6q3hsPecAbkj1EVPMT8UAkdJkVOYaoXigysNpvmpn3xBtJuuqVn9waDAWfddV/g7s5QMnYUyDo74S2mh0ayXMUTCbTkdP6vLW6p21sxBALu7vCnSybVMx8VJFVhrz3vViApc5/YafuN6HbQR9/kehdkGzHPzLOmB0uj4UcOODVgoIC5aeTJ/oWlBTe8/Dhj/+My8g8n5EkIxU40Obwv8yL/34azGZkYHPgREqcD303CBZCrVRHvYLgGZNjzedXH1decwT5CAMj7r/qZMQigycs92/8zXfO32cHVTPjGEbAEPQm+n/I/2h7Y5DD2aA7BeM5BmAhQKSf2tgOURkA1/vQMXUPOTAoBR1Wcd429ifzfKuGnlhUnNK9dtrpUsa/ePGIKyKfG51rNEwoMPVAffvFvC7wm/ERlw/G2gBmsxk83oAicvIhyUyGG7fDz3RuIBTB5ny/5N2tB21nXlAU8kGbLG/NCromN4slZ74x9LPPLj1wjjG1Y2MGzLijU3JfDYbtNVVEUlbjAaHWaFDlJ9oRqW3Lu/8J3NgfQfBNfcNbceF/tGuRVg9p/e+TQwObtgHjpz7lQVckcwFpw6Mg7/038NPuA/GqNSCtuQ+UihX67YMrQO2sTO6olAPAlp4P/IwHNcAnBAPTaK1tCIgXv
```

Comment:
thoughts on loading this string from another file?

Reply:
Definitely something worth considering, I embedded it because I made this a [Tampermonkey Script](https://github.com/Krakaw/tampermonkey/blob/master/aws_favicon.user.js)

Reply:
I've now removed the base64 entirely, it just loads the currently used sprite automatically on load.